vulnerabilities/tests/test_view.py

```diff
@@ -57,6 +57,12 @@ def test_package_view(self):
         self.assertEqual(len(pkgs), 1)
         self.assertEqual(pkgs[0].purl, "pkg:nginx/nginx@1.0.15")
 
+    def test_package_view_with_purl_fragment(self):
```

Comment:
We need many more tests IMHO to assert the search results quality before and ensure this does not regress.

Reply:
@pombredanne we already had tests for the previous implementation and none of them fail, please tell me any particular corner case that should we check for ?